modules/offload-s3/module.go

```diff
@@ -284,14 +284,18 @@ func (m *Module) Upload(ctx context.Context, className, shardName, nodeName stri
 // cloud provider (S3, Azure Blob storage, Google cloud storage)
 // {dataPath}/{className}/{shardName}/{content}
 func (m *Module) Download(ctx context.Context, className, shardName, nodeName string) error {
+	localPath := fmt.Sprintf("%s/%s/%s", m.DataPath, strings.ToLower(className), shardName)
+	return m.DownloadToPath(ctx, className, shardName, nodeName, localPath)
+}
+
+func (m *Module) DownloadToPath(ctx context.Context, className, shardName, nodeName, localPath string) error {
```

Comment:
@moogacs any concerns about this change?

Reply:
IMO It's OK